guidelines/monthly.md

```diff
@@ -1 +1,26 @@
-TBD
+# BarcelonaJS Monthly events guidelines.
+
+About
+
+- BarcelonaJS will celebrate one event per month
+- The event is hosted by an external entity/company
+- The event should start around 19:00
+- The event must be free
+- BarcelonaJS won't charge anything for organizing the event.
+- All participants and organizers should agree with our CoC
+- The external entity can do an intro talk limited to 5m
+
+Talks:
+
+- Talks must be in English
+- Talks should be in the 20-90m range.
+- Talks should be javascript related.
```

Comment:
I would rephrase this to javascript / frontend related. We had in the past talks about CSS / SaSS.